### ISSUE TITLE
EditMessageCaptionArgs could not be constructed

### ISSUE THREAD

**Issue by pavel-zhur** [Eptagone/Telegram.BotAPI#20]
It says, `this.MessageId = this.MessageId ?? throw new ArgumentNullException`.

```

    /// <summary>
    /// Initialize a new instance of <see cref="T:Telegram.BotAPI.UpdatingMessages.EditMessageCaptionArgs" />.
    /// </summary>
    /// <param name="chatId">Unique identifier for the target chat or username of the target channel (in the format @channelusername).</param>
    /// <param name="messageId">Identifier of the message to edit.</param>
    /// <exception cref="T:System.ArgumentNullException"></exception>
    public EditMessageCaptionArgs(long chatId, int messageId)
    {
      this.ChatId = (object) chatId;
      this.MessageId = this.MessageId ?? throw new ArgumentNullException(nameof (MessageId));
    }

    /// <summary>
    /// Initialize a new instance of <see cref="T:Telegram.BotAPI.UpdatingMessages.EditMessageCaptionArgs" />.
    /// </summary>
    /// <param name="chatId"></param>
    /// <param name="messageId">Identifier of the message to edit.</param>
    /// <exception cref="T:System.ArgumentNullException"></exception>
    public EditMessageCaptionArgs(string chatId, int messageId)
    {
      this.ChatId = (object) (chatId ?? throw new ArgumentNullException(nameof (chatId)));
      this.MessageId = this.MessageId ?? throw new ArgumentNullException(nameof (MessageId));
    }

```

**Comment by Eptagone** [comment]
It's true! I'll fix it with the next release.
Sorry for the trouble.

**Comment by Eptagone** [comment]
Done. Fixed with commit 3ab62f20d8a4fc2433ff5b262775d5544d9b433a
